content/guides/java/tracers.md

```diff
@@ -2,7 +2,67 @@
 title: "Java: Tracers"
 ---
 
-* Setting up your tracer
-* Using the Global Tracer
-* Accessing the Active Span
-* Accessing tracer specific features.
\ No newline at end of file
+#### Setting up your Tracer
+
+Different **Tracer** implementations vary in how and what parameters they receive at initialization time, such as:
+
+* Component name for this application's traces.
+* Tracing endpoint.
+* Tracing credentials.
+* Sampling strategy.
+
+
+For example, initializing the **Tracer** implementation of `Jaeger` looks like this:
```

Comment:
s/looks like this/might look like this/

The "might" is because it could be also as simple as `Configuration.fromEnv().getTracer()`

Reply:
> might look like this

:) Makes sense, definitely.